### PR TITLE
Use more portable invocation of base64 tool

### PR DESCRIPTION
**Pull request by ayushpatil2122**
### Description

```
kubectl get csr my-svc.my-namespace -o json | \
  jq '.status.certificate = "'$(base64 ca-signed-server.pem | tr -d '\n')'"' | \
  kubectl replace --raw /apis/certificates.k8s.io/v1/certificatesigningrequests/my-svc.my-namespace/status -f -

```
During the "Upload the signed certificate" step (https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/#upload-the-signed-certificate) the command didn't work without the "-i" option using the base64 command.

it had to be

```
kubectl get csr my-svc.my-namespace -o json | \
  jq '.status.certificate = "'$(base64 -i ca-signed-server.pem | tr -d '\n')'"' | \
  kubectl replace --raw /apis/certificates.k8s.io/v1/certificatesigningrequests/my-svc.my-namespace/status -f -
```

### Issue
#48586

### CODE REVIEW

**Comment by thisisharrsh** [comment]
_Deployment preview :: https://deploy-preview-48591--kubernetes-io-main-staging.netlify.app/docs/tasks/tls/managing-tls-in-a-cluster/#upload-the-signed-certificate_

**Comment by thisisharrsh** [comment]
As mentioned in the comment (https://github.com/kubernetes/website/issues/48586#issuecomment-2445123310), It will be better if we could add a note.

**Comment by sftim** [comment]
/retitle Use more portable invocation of base64 tool

File: content/en/docs/tasks/tls/managing-tls-in-a-cluster.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ Finally, populate the signed certificate in the API object's status:
 
 ```shell
 kubectl get csr my-svc.my-namespace -o json | \
-  jq '.status.certificate = "'$(base64 ca-signed-server.pem | tr -d '\n')'"' | \
+  jq '.status.certificate = "'$(base64 -i ca-signed-server.pem | tr -d '\n')'"' | \
```
Comment (stmcginnis) [inline content/en/docs/tasks/tls/managing-tls-in-a-cluster.md:265]: With the BSD version of base64 included with macOS, `-i` means "input file". It does not take a positional input file argument.
With the gnu version in most linux distros, `-i` is short for `--ignore-garbage`. It _does_ allow for a positional input file argument.

There shouldn't be any "garbage" in the pem file, but it looks safe to use it to be compatible with either implementation.

**Pull request closed without merging**